presto-kudu/src/test/java/io/prestosql/plugin/kudu/TestKuduIntegrationSmoke.java

```diff
@@ -136,5 +139,6 @@ public final void destroy()
     {
         assertUpdate("DROP TABLE " + ORDERS.getTableName());
         getQueryRunner().close();
+        kuduServer.close();
```

Comment:
Let's move this method above and fix it by removing the redundant query runner close and the drop table. (no point in dropping since this is a temporary server we are about to destroy)